composer.json

```diff
@@ -37,18 +37,19 @@
         }
     },
     "require-dev": {
+        "friendsofphp/php-cs-fixer": "^2.4",
+        "jakub-onderka/php-parallel-lint": "^0.9.2",
+        "phpro/grumphp": "^0.11.6",
         "phpunit/phpunit": "^5.7",
-        "friendsofphp/php-cs-fixer": "^2.4"
+        "sensiolabs/security-checker": "^4.1"
     },
     "bin": [
         "bin/bunq-install"
     ],
-    "scripts": {
-        "phpcs": [
-            "./vendor/bin/php-cs-fixer fix -v --dry-run --using-cache=no src"
-        ],
-        "phpcs-fix": [
-            "./vendor/bin/php-cs-fixer fix -v --using-cache=no src"
-        ]
+    "config": {
+        "sort-packages": true,
+        "platform": {
+            "php": "5.6"
```

Comment:
Yes.

1. There will be no longer problems as phpstan/phpstan in (7e8fad5). When someone now try to install an packages which has no support for PHP 5.6 they will get an Composer error

**_composer require phpstan/phpstan_**
```
[InvalidArgumentException]                                                               
Could not find package phpstan/phpstan at any version matching your PHP version 5.6.0.0  
```

2. Now I don't need to change my local environment to PHP 5.6 when I add/update some PHP packages.